src/plugins/discover/public/context_awareness/profile_providers/example_data_source_profile/profile.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiBadge } from '@elastic/eui';
+import type { DataTableRecord } from '@kbn/discover-utils';
+import { isOfAggregateQueryType } from '@kbn/es-query';
+import { getIndexPatternFromESQLQuery } from '@kbn/esql-utils';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { capitalize } from 'lodash';
+import React from 'react';
+import { DataSourceType, isDataSourceType } from '../../../../common/data_sources';
+import { DataSourceCategory, DataSourceProfileProvider } from '../../profiles';
+
+export const exampleDataSourceProfileProvider: DataSourceProfileProvider = {
+  profileId: 'example-data-source-profile',
+  profile: {
+    getCellRenderers: (prev) => () => ({
+      ...prev(),
+      'log.level': (props) => {
+        const level = getFieldValue(props.row, 'log.level');
+
+        if (!level) {
+          return <span css={{ color: euiThemeVars.euiTextSubduedColor }}>(None)</span>;
```

Comment:
I suggest to add `data-test-subj` here as well, since this is also a customized renderer and not the original one.

Reply:
Makes sense to me, added here: https://github.com/elastic/kibana/pull/185905/commits/de21734c103a1a4761566aa76101171a6c29e3cc